scripts/proposals/AtlasFeesAIP/atlais-aip-data.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
file name seems to be typoed `scripts/proposals/AtlasFeesAIP/atlas-aip-data.json`